### DISCUSSION TITLE
UpdateManager warning: The operation was canceled.

### DISCUSSION THREAD

**Discussion by Katzenkralle** [Pryaxis/TShock#1947] [Q&A] [answered]
Hello,
I have had the following error every few minutes since yesterday:
UpdateManager warning: The operation was canceled.
	-> No such host is known
Retrying in 5 minutes.
![Unbenannt](https://user-images.githubusercontent.com/65792561/83237767-b8273e00-a195-11ea-88fb-8b0e72e4c226.PNG)

**Comment by hakusaro** [comment, marked as answer]
This is normal -- you're probably blocking our update server or something. Nothing to worry about, just unable to check for updates.

**Reply by nathan-alden-sr** [reply]
Can you tell me what the server hostname is? I'm also experiencing this on my Ubuntu server, even though I don't have any specific firewall rules on that server. I do run a Pi-hole for network-level ad-blocking, though, so I can whitelist whatever your update server hostname is.

**Comment by hakusaro** [comment]
It's update.tshock.co.

On July 10, 2022, GitHub ***@***.***> wrote:
> Can you tell me what the server hostname is? I'm also experiencing
> this on my Ubuntu server, even though I don't have any specific
> firewall rules on that server. I do run a Pi-hole for network-level
> ad-blocking, though, so I can whitelist whatever your update server
> hostname is.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/Pryaxis/TShock/discussions/1947#discussioncomment-
3117383>, or unsubscribe <https://github.com/notifications/unsubscribe-
auth/AAECBJ6J67ZUM55H3PEMC2LVTNDJJANCNFSM4NN2P4ZQ>.
You are receiving this because you commented.Message ID:
***@***.***>
